internal/cli/config/config.go

```
@@ -0,0 +1,115 @@
+		return err
+	}
+	if err := os.MkdirAll(cfgDir, 0o755); err != nil {
+		return fmt.Errorf("mkdir %s: %w", cfgDir, err)
+	}
+	cfgPath, err := orchpaths.ConfigFile()
+	if err != nil {
+		return err
+	}
+	if !force {
+		if _, err := os.Stat(cfgPath); err == nil {
+			fmt.Fprintf(w, "config already exists: %s (use --force to overwrite)\n", cfgPath)
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)